jupyterlab-dataops/Dockerfile

```diff
@@ -34,16 +40,69 @@ RUN pip install -r /opt/requirements/requirements.txt
 COPY resources/logos/ /opt/logos/
 
 # jupyter config
-COPY resources/jupyter/ /etc/jupyter/
+COPY resources/jupyter/gcp/ /etc/jupyter/
 
 # Expose port for Vertex AI compatibility
 EXPOSE 8080
 
-FROM base as bigquery
+FROM base as gcp
 COPY requirements/requirements-bigquery.txt /opt/requirements/
 RUN pip install -r /opt/requirements/requirements-bigquery.txt
 ADD resources/cloudbeaver/conf/initial-data-sources-bigquery.conf.j2 /opt/cloudbeaver/conf/initial-data-sources.conf.j2
 COPY resources/cloudbeaver/set_metadata.py /opt/cloudbeaver/set_metadata.py
 COPY resources/cloudbeaver/set_metadata_run_jupyter.sh /set_metadata_run_jupyter.sh
 RUN chmod +x /set_metadata_run_jupyter.sh
 CMD ["/set_metadata_run_jupyter.sh"]
+
+# Jupyterhub version
+FROM jupyter/minimal-notebook:python-3.8.13 as jupyterhub
```

Comment:
Looks like there is a lot of duplication,if it is not possible to reuse the code maybe it is better to have it in separate Dockerfile?